### PR TITLE
Promote customSecurityAttributeDefinition from beta to v1.0

### PR DESCRIPTION
**Pull request by CecilyK**
Tracked via this content review work item [86489](https://dev.azure.com/msft-skilling/Content/_queries/edit/86489/?triage=true) in the Skilling ADO.

### CODE REVIEW

**Comment by FaithOmbongi** [comment]
Hi @CecilyK - Please create a content review work item @ https://aka.ms/msgraph-identity-workitem and share the required info in the work item description. Thank you.

File: api-reference/beta/resources/custom-security-attributes-overview.md [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
 ms.localizationpriority: medium
 ms.prod: "directory-management"
 doc_type: conceptualPageType
-ms.date: 02/14/2023
 ---
 
-# Overview of custom security attributes using the Microsoft Graph API (Preview)
+# Overview of custom security attributes using the Microsoft Graph API
 
 > [!IMPORTANT]
```
Comment (Jarbas-MSFT) [inline api-reference/beta/resources/custom-security-attributes-overview.md:12]: Do we still need this note here? Please, remove or update this note.

**Comment by JarbasHorst** [comment]
Hello @CecilyK, I reviewed your PR and left one comment. Please, take a look when you will have time.

File: api-reference/beta/resources/custom-security-attributes-overview.md (hunk 1 above)
Comment (CecilyK) [reply api-reference/beta/resources/custom-security-attributes-overview.md]: @Jarbas-MSFT We do not need the note. I've removed it.

File: api-reference/beta/api/allowedvalue-get.md [hunk 2]
```diff
@@ -8,13 +8,15 @@ doc_type: apiPageType
 ---
```
Comment (JarbasHorst) [inline api-reference/beta/api/allowedvalue-get.md:8]: Hi @CecilyK, I noticed this entry outside of the Microsoft Graph repository that I think you should also update: https://learn.microsoft.com/en-us/azure/role-based-access-control/conditions-overview#status-of-condition-features

File: concepts/custom-security-attributes-examples.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ ms.prod: "directory-management"
 ms.date: 02/14/2023
 ---
 
-# Manage custom security attribute assignments (preview)
+# Manage custom security attribute assignments
 
 > [!IMPORTANT]
```
Comment (JarbasHorst) [inline concepts/custom-security-attributes-examples.md:15]: Do we still need this note? Please, remove or update this note.

Comment (CecilyK) [reply concepts/custom-security-attributes-examples.md]: Removed

**Comment by Jarbas-MSFT** [comment]
Hi @Lauragra, hi @FaithOmbongi, this PR is ready for final review;

File: api-reference/beta/api/allowedvalue-get.md (hunk 2 above)
Comment (CecilyK) [reply api-reference/beta/api/allowedvalue-get.md:8]: This will be updated on the azure-docs-pr repo by @rolyon

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Hi @Jarbas-MSFT - Thanks for the review. Everything looks good, just two minor edits.

@CecilyK - I've left two comments, please address them. Before approving, I'll also push a commit to move the RBAC information into include files for ease of maintenance.

File: concepts/custom-security-attributes-examples.md
[the remark this answers is not in the document]
Comment (CecilyK) [reply concepts/custom-security-attributes-examples.md:1]: This has been updated.

**Comment by CecilyK** [comment]
#feedback-addressed

**Comment by CecilyK** [comment]
> Hi @CecilyK - I've pushed a few commits to use include files for the RBAC information.
> 
> Approved.

Thanks @FaithOmbongi

**Pull request merged**